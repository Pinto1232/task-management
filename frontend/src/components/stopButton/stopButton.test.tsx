import {render, screen} from '@testing-library/react'
import StopButton from './stopButton'

describe('StopButton', () => {
   test('should render without crashing', () => {
    render(<StopButton>Content</StopButton>)
    const el = screen.getByTestId('stop-button');
    expect(el).toBeInTheDocument();
    expect(el).toHaveTextContent('Content');
    })
   test('should match snapshot', () => {
    const {asFragment} = render(<StopButton>Content</StopButton>)
        expect(asFragment()).toMatchSnapshot()
    });
});
