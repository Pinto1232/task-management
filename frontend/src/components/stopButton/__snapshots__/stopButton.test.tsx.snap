// Jest Snapshot v1, https://jestjs.io/docs/snapshot-testing

exports[`StopButton should match snapshot 1`] = `
<DocumentFragment>
  <div
    class="container"
    data-testid="stop-button"
  >
    Content
  </div>
</DocumentFragment>
`;
